### ISSUE TITLE
CircleCI upgrade

### ISSUE THREAD

**Issue by NicolasGensollen** [nilearn/dev-days-2021#7]
Check if we can upgrade CircleCI plan for the duration of the sprint. 
Otherwise (and maybe even if we have a better plan...), we will need to find a way to skip doc building for trivial PRs.
I can ask people from MNE how they are handling this in their current "beginners" sprint.

**Comment by bthirion** [comment]
Yes, thx ! Keep me posted on that. We can dedicate a bit of money to that if necessary.
